in_toto/runlib.go

```diff
@@ -240,6 +240,10 @@ command execution.
 */
 func RunCommand(cmdArgs []string, runDir string) (map[string]interface{}, error) {
 
+	if cmdArgs == nil || len(cmdArgs) == 0 {
```

Comment:
It is actually not that easy.. this would lead to an error if cmdArgs is empty.
The in-toto python implementation allows empty cmdArgs, hence we need a fix in both functions. Like here:
https://github.com/in-toto/in-toto-golang/pull/126/files